Fix/Lint Inspec Verify Playbook

Lirt opened the pull request:
#### PR Type

- Bugfix Pull Request

Lint `Inspec` playbook `verify.yml` with `ansible-lint`.

Tested with molecule `2.19`.

decentral1se commented:
Thanks @Lirt! What was the issue `ansible-lint` was reporting? 

For another PR, but maybe we should lint our internal plays as well!

Is this being tested btw? Or do you have some ideas? Manual test logs work as well ...

Lirt commented:
Hi, commit messages have it in description, but here is full `ansible-lint` report:

```bash
ansible-lint verify.yml
[403] Package installs should not use latest
verify.yml:57
Task/Handler: Install Inspec (yum)

[301] Commands should not change things if nothing needs doing
verify.yml:75
Task/Handler: Register test files

[301] Commands should not change things if nothing needs doing
verify.yml:79
Task/Handler: Execute Inspec tests
```

About automated testing, we can use `molecule` itself :D, but this is always hard, because we need some infrastructure for it. 

Test log:
```bash
    PLAY [Verify] ******************************************************************
    
    TASK [Gathering Facts] *********************************************************
    ok: [molecule-test-ci-runner-0]
    
    TASK [Setting variables (CentOS 6 / RHEL 6)] ***********************************
    skipping: [molecule-test-ci-runner-0]
    
    TASK [Setting variables (CentOS 7 / RHEL 7)] ***********************************
    skipping: [molecule-test-ci-runner-0]
    
    TASK [Setting variables (Debian 8 / Ubuntu 14.04)] *****************************
    skipping: [molecule-test-ci-runner-0]
    
    TASK [Setting variables (Debian 8 / Ubuntu 16.04)] *****************************
    ok: [molecule-test-ci-runner-0]
    
    TASK [Setting variables (Debian 9 / Ubuntu 18.04)] *****************************
    skipping: [molecule-test-ci-runner-0]
    
    TASK [Download Inspec] *********************************************************
    changed: [molecule-test-ci-runner-0]
    
    TASK [Install Inspec (apt)] ****************************************************
    changed: [molecule-test-ci-runner-0]
    
    TASK [Install Inspec (yum)] ****************************************************
    skipping: [molecule-test-ci-runner-0]
    
    TASK [Create Molecule directory for test files] ********************************
    changed: [molecule-test-ci-runner-0]
    
    TASK [Copy Inspec tests to remote] *********************************************
    changed: [molecule-test-ci-runner-0] => (item=/home/ovasko/Projects/exponea/ansible/roles/ci-runner/molecule/default/tests/test_runner.rb)
    
    TASK [Execute Inspec tests] ****************************************************
    ... VERY LONG TEXT ...
    TASK [Display details about the Inspec results] ********************************
    ... VERY LONG TEXT ...
    TASK [Fail when tests fail] ****************************************************
    ... VERY LONG TEXT ...
```

decentral1se commented:
Ah! Nice, I raised https://github.com/ansible/molecule/issues/1867 for what I meant about `ansible-lint` running on internal plays. There seems to be some further issue to resolve to get that working right now ...

I restarted the CI build for this!
